### ISSUE TITLE
Return the return code from shell

### ISSUE THREAD

**Issue by pablochacin** [jbuck/assume-aws-role#18]
When using the assume-aws-role command as part of a script, it would be convenient to receive the   return code from the shell spawn instead of always returning 0.

Example:
`echo "aws s3 sync ... " | assume-aws-role myrole`

will always return 0, even if the aws s3 command fails.

**Comment by pablochacin** [comment]
The changes in the code are pretty simple:

```
--- a/bin/assume-aws-role.js
+++ b/bin/assume-aws-role.js
@@ -140,8 +140,13 @@ STS.assumeRole({
   modEnv.PS1 = "(assume-aws-role " + command + ")$ ";
   modEnv.ASSUME_AWS_ROLE = command;
 
-  spawn(process.env.SHELL, {
+  var sh = spawn(process.env.SHELL, {
     env: modEnv,
     stdio: "inherit"
   });
+
+  sh.on('close',(code)=>{
+       process.exit(code);
+  });
+
 });
```

**Comment by jbuck** [comment]
Yeah, this seems like a good change. Could you throw this in a PR and I'll merge it? Thanks!

**Comment by pablochacin** [comment]
Done

**Comment by jbuck** [comment]
Closed by #19
